types/setting.go

```diff
@@ -1482,6 +1485,16 @@ var (
 			ValueIntRangeMinimum: 1,
 		},
 	}
+
+	SettingDefinitionEnableShareManagerFastFailover = SettingDefinition{
+		DisplayName: "Enable Share Manager Fast Failover",
+		Description: "Turn on logic to detect and move stale RWX volumes quickly (Experimental)",
+		Category:    SettingCategoryDangerZone,
```

Comment:
I am currently understand that a setting in danger zone is a setting when apply will restart some critical components like IM pod which can crash all other running Longhorn volume. Looks like this new setting is not like that. It will not crash anyone when it is enable/disable

Reply:
Fixed, thank you.